src/libs/transforms.ts

```diff
@@ -66,47 +81,70 @@ export interface FileRecord {
 }
 
 export function subscriptionsDataFileToSubscriptions(file: string): ZuoraSubscription[] {
-  const splitLines = str => str.split(/\r?\n/);
-  const lines = splitLines(file);
-  lines.shift();
-  const subscriptions = lines.map((line) => {
-    const elements = line.split(",");
+
+  console.log("Zuora subscriptions data file:");  
+  console.log(file);
+
+  const headers = [
+    'Subscription.Name', 
+    'Subscription.DeliveryAgent__c', 
+    'SoldToContact.Address1', 
+    'SoldToContact.Address2', 
+    'SoldToContact.City', 
+    'SoldToContact.PostalCode', 
+    'SoldToContact.FirstName', 
+    'SoldToContact.LastName', 
+    'SoldToContact.SpecialDeliveryInstructions__c', 
+    'RatePlanCharge.Quantity', 
+    'RatePlanCharge.Name'
+  ];
+
+  const records: string[][] = parse(file);
+  console.log("records:");
+  console.log(JSON.stringify(records));
+
+  records.shift();
```

Comment:
I'm surprised if there's not an ignore-header flag you can pass in

Reply:
According to the documentation, there is one, but only using custom column names.